packages/darklang/languageTools/lsp-server/semanticTokens.dark

```diff
@@ -26,7 +26,8 @@ module Darklang =
             "parameter" // [7] for function parameter identifiers
             "variable" // [8] for general identifiers
             "function" // [9] for function names/identifiers
-            "property" ] // [10] for field names
+            "property" // [10] for field names
+            "enumCase" ] // [11] for enum case names
```

Comment:
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#semanticTokenTypes I think `enumMember` may be the more universal term. (using the universal term should help various VS Code themes style our code correctly)